### PR TITLE
870/amends to qt dl

### PR DESCRIPTION
**Pull request by tomlovesgithub**
There is some pretty evil logic in the responses view.
Hopefully ive managed to target the individual question scores object correctly, unfortunately i couldn't find a existing response with them, if not the lines where they are inserted into the sheet are hopefully clear enough

### CODE REVIEW

File: src/views/Exercises/Tasks/QualifyingTests/QualifyingTest/Responses.vue [hunk 1]
```diff
@@ -120,18 +137,25 @@ export default {
...
             if (element.testQuestions.questions[index].response && (element.testQuestions.questions[index].response.selection !== undefined)) { 
-              if (this.qualifyingTest.testQuestions.questions[index].options[element.testQuestions.questions[index].response.selection.mostAppropriate] !== undefined) {
-                row.push(this.qualifyingTest.testQuestions.questions[index].options[element.testQuestions.questions[index].response.selection.mostAppropriate].answer);
+              if (this.qualifyingTest.testQuestions.questions[index].options[element.testQuestions.questions[index].response.selection.mostAppropriate] !== undefined && this.qualifyingTest.testQuestions.questions[index].options[element.testQuestions.questions[index].response.selection.leastAppropriate] !== undefined) {
+                row.push(
+                  this.qualifyingTest.testQuestions.questions[index].options[element.testQuestions.questions[index].response.selection.mostAppropriate].answer,
+                  this.qualifyingTest.testQuestions.questions[index].options[element.testQuestions.questions[index].response.selection.leastAppropriate].answer,
```
Comment (tomlovesgithub) [inline src/views/Exercises/Tasks/QualifyingTests/QualifyingTest/Responses.vue:143]: this stuff ^^ pure evil

Comment (tomlovesgithub) [reply src/views/Exercises/Tasks/QualifyingTests/QualifyingTest/Responses.vue:143]: (hard to understand it, is there anyway we can sweeten this up?)

**Comment by tomlovesgithub** [comment]
Jeez this git history is awful sorry guys, happens every time i make an effort to not have an awful git history it happens, sorry!

**Review by lloback — APPROVED** [review]
I am going to approve this.
It is not showing any errors, but the data on DEV does not allow me to test this properly. Will review on staging.

**Pull request merged**